docker/docker.mk

```diff
@@ -130,3 +131,34 @@ else
 	$(DOCKER) buildx build . $(DOCKER_BUILD_ARGS) --file $(DOCKERFILE_DIR)/Dockerfile-tools --platform $(BUILDX_PLATFORMS) --tag ${TOOL_IMG}:${VERSION} --push
 endif
 endif
+
+.PHONY: build-charts-image
+build-charts-image: DOCKER_BUILD_ARGS += --cache-to type=gha,mode=max,scope=${GITHUB_REF_NAME}-charts-image --cache-from type=gha,scope=${GITHUB_REF_NAME}-charts-image
```

Comment:
expected update to addon controller for creating helm install job that use this container as initContainer and `cp` chart tgz files to a shared volume

Reply:
updated [ce81f4f](https://github.com/apecloud/kubeblocks/pull/4219/commits/ce81f4fa0458237a789ca11565663db4a9e5eb11)